client/src/components/common/Checkbox/index.css

```diff
@@ -0,0 +1,36 @@
+.checkbox__home{
+  display: inline-flex;
+  margin: 10px;
+}
+
+.checkbox__input[type="checkbox"] {
+  display: none;
+}
+
+.checkbox_span{
+  display: inline-block;
+  background: #ffffff;
+  width: 20px;
+  height: 20px;
+  border-radius: 50%;
+  vertical-align:middle;
+  margin:-2px 10px 0 0; 
+}
+
+.checkbox_label{
+  color: #ffffff;
+  padding: 10px 30px;
+  background: #3873B3;
+  border-radius: 16px;
+  cursor: pointer;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.checkbox_label .checkbox__input[type="checkbox"]:checked {
+  background: #52DE97;
+}
+
+.checkbox_label .checkbox_span .checkbox__input[type="checkbox"]:checked {
```

Comment:
attribute selector

Reply:
another thing, I don't think these selectors will match anything - the `checkbox__input` element is not within the `checkbox_span` but next to it, thus these styles will never be applied.. pls verify this and either remove or fix it